bin/update_lambda_fcn.py

```diff
@@ -35,6 +35,7 @@
 from lib import zip
```

Comment:
Does it make sense to combind all of the lambda scripts into a single one? Just wondering about overpopulating the bin/ directory.

Reply:
Potentially. Future task?